Modules/Loadable/Markups/VTKWidgets/vtkSlicerAngleRepresentation3D.cxx

```diff
@@ -69,6 +69,20 @@ vtkSlicerAngleRepresentation3D::vtkSlicerAngleRepresentation3D()
   this->ArcActor->SetMapper(this->ArcMapper);
   this->ArcActor->SetProperty(this->GetControlPointsPipeline(Unselected)->Property);
 
+  this->LineOccludedMapper = vtkSmartPointer<vtkPolyDataMapper>::New();
```

Comment:
It would be more consistent to keep the same order (all actors then all mappers), and it would be better to keep ...Mapper and ...OccludedMapper closer to each other everywhere, because if we want to add a mapper then we most likely need to add an OccludedMapper as well.